client/apps/edit/components/layout/index.coffee

```diff
@@ -156,18 +156,26 @@ module.exports = class EditLayout extends Backbone.View
 
   checkSeo: =>
     imageCount = 0
+    textIndex = 0
     @fullText = []
-    @fullText.push(" <p> " + $(@article.get("lead_paragraph")).text() + " </p> ")
-    for section in $(@article.get("sections"))
-      if section.type is "text"
+    if ($(@article.get('lead_paragraph')).text() != '')
+      @fullText.push(' <p> ' + $(@article.get('lead_paragraph')).text() + ' </p> ')
+      textIndex += 1
+
+    for section in $(@article.get('sections'))
+      if section.type is 'text' && textIndex == 0
+        @fullText.push(' <p> ' + $(section.body).find('p').andSelf().filter('p:first').text() + ' </p> ')
+        @fullText.push($(section.body).find('p').andSelf().filter('p:not(:first)').text())
```

Comment:
wish there was an easier way to do this! if anything, we should break out these lines so we can read them better.
```
@fullText.push(
   $(section.body)
      .find('p')
      .andSelf()
      .filter('p:not(:first)')
      .text()
)
```
syntax above might be slightly off so test to make sure it's works!